root/components/forms.tt

```diff
@@ -281,6 +281,7 @@
         <option value="French">[% l('French') %]</option>
         <option value="Turkish">[% l('Turkish') %]</option>
       </select>
+      (<a href="/doc/Guess_Case" target="_blank" class="open">[% l('help') %]</a>)
```

Comment:
I don't see any `open` class in our CSS that'd apply here.

Reply:
I don’t recall why I used this class, removed from https://github.com/metabrainz/musicbrainz-server/pull/1179/commits/ee0f533c7ecfb8e664a23ef076835f1adbd050be.